src/main/java/io/mosip/print/util/CbeffToBiometricUtil.java

```diff
@@ -0,0 +1,335 @@
+	 * @return the photo
+	 * @throws Exception
+	 *             the exception
+	 */
+	public byte[] getImageBytes(String cbeffFileString, String type, List<String> subType) throws Exception {
+		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(), "",
+				"CbeffToBiometricUtil::getImageBytes()::entry");
+
+		byte[] photoBytes = null;
+		if (cbeffFileString != null) {
+			List<BIRType> bIRTypeList = getBIRTypeList(cbeffFileString);
+			photoBytes = getPhotoByTypeAndSubType(bIRTypeList, type, subType);
+		}
+		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(), "",
+				"CbeffToBiometricUtil::getImageBytes()::exit");
+
+		return photoBytes;
+	}
+
+	/**
+	 * Gets the photo by type and sub type.
+	 *
+	 * @param bIRTypeList
+	 *            the b IR type list
+	 * @param type
+	 *            the type
+	 * @param subType
+	 *            the sub type
+	 * @return the photo by type and sub type
+	 */
+	private byte[] getPhotoByTypeAndSubType(List<BIRType> bIRTypeList, String type, List<String> subType) {
+		byte[] photoBytes = null;
+		for (BIRType birType : bIRTypeList) {
+			if (birType.getBDBInfo() != null) {
+				List<SingleType> singleTypeList = birType.getBDBInfo().getType();
+				List<String> subTypeList = birType.getBDBInfo().getSubtype();
+
+				boolean isType = isSingleType(type, singleTypeList);
+				boolean isSubType = isSubType(subType, subTypeList);
+
+				if (isType && isSubType) {
+					photoBytes = birType.getBDB();
+					break;
+				}
+			}
+		}
+		return photoBytes;
+	}
+
+	/**
+	 * Checks if is sub type.
+	 *
+	 * @param subType
+	 *            the sub type
+	 * @param subTypeList
+	 *            the sub type list
+	 * @return true, if is sub type
+	 */
+	private boolean isSubType(List<String> subType, List<String> subTypeList) {
+		return subTypeList.equals(subType) ? Boolean.TRUE : Boolean.FALSE;
+	}
+
+	/**
+	 * Checks if is single type.
+	 *
+	 * @param type
+	 *            the type
+	 * @param singleTypeList
+	 *            the single type list
+	 * @return true, if is single type
+	 */
+	private boolean isSingleType(String type, List<SingleType> singleTypeList) {
+		boolean isType = false;
+		for (SingleType singletype : singleTypeList) {
+			if (singletype.value().equalsIgnoreCase(type)) {
+				isType = true;
+			}
+		}
+		return isType;
+	}
+
+	/**
+	 * Merge cbeff.
+	 *
+	 * @param cbeffFile1
+	 *            the cbeff file 1
+	 * @param cbeffFile2
+	 *            the cbeff file 2
+	 * @return the input stream
+	 * @throws Exception
+	 *             the exception
+	 */
+	public InputStream mergeCbeff(String cbeffFile1, String cbeffFile2) throws Exception {
+		byte[] mergedCbeffByte = null;
+		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(), "",
+				"CbeffToBiometricUtil::mergeCbeff()::entry");
+
+		/*
+		 * if(!checkFileVersions(cbeffFile1, cbeffFile2)) { throw new
```

Comment:
Please remove unused commentted code here. We should document what's exceptions might throws from this method.